fetch_rapids.cmake

```diff
@@ -1,5 +1,5 @@
 # =============================================================================
-# Copyright (c) 2018-2023, NVIDIA CORPORATION.
+# Copyright (c) 2018-2024, NVIDIA CORPORATION.
```

Comment:
This probably can be reverted?

Reply:
Yes, this is a leftover when customizing the cuco git tag, will be reverted.

Reply:
Done